src/plugins/vis_types/vega/public/vega_view/vega_base_view.js

```diff
@@ -49,6 +49,30 @@ export function bypassExternalUrlCheck(url) {
   return { url, bypassToken };
 }
 
+const externalUrlsAreNotEnabledError = new Error(
```

Comment:
it's not cool to keep instance of error. Please move it to place where you trigger it or wrap in into function e.g. 

`const getExternalUrlsAreNotEnabledError =  () =>new Error(`